### ISSUE TITLE
Feature Request: Support foldable phone

### ISSUE THREAD

**Issue by Arc-Jung** [crackededed/Xtra#136]
![화면 캡처 2022-08-20 190230](https://user-images.githubusercontent.com/43470915/185740579-439a391b-78b8-4fac-9261-8c1b47ec38b7.png)
I want bottom chating feature in foldable phone.

**Comment by Arc-Jung** [comment]
![Uploading Screenshot_20220930_154621.png…]()
@crackededed 
I want bottom chat feature. like Frosty(https://github.com/tommyxchow/frosty)

**Comment by Nitrousoxide** [comment]
I agree that this would be helpful.  Having it identify where the letterboxing is being drawn and placing the chat there would generally be nice to more efficiently make use of the screen real estate.
